src/lf_imap.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python
```

Comment:
If we want to maintain our 100% bash codebase, apparently there's [this project](https://github.com/dominictarr/JSON.sh) which implements a JSON parser in bash 😂